README.rst

```diff
@@ -7,6 +7,181 @@ OpenWISP Notifications Module
 .. image:: https://coveralls.io/repos/github/openwisp/openwisp-notifications/badge.svg?branch=master
    :target: https://coveralls.io/github/openwisp/openwisp-notifications?branch=master
 
-----------------------
+------------
 
-WIP
+Abstracts code for generation and transmission of notifications in OpenWISP 2. 
```

Comment:
I'd put some more effort in this first line.

Reply:
Something like "This module provides email and web notifications for OpenWISP, it's used to notify OpenWISP users about meaningful events in their network."